samples/csharp/web/avatar/Controllers/AvatarController.cs

```diff
@@ -0,0 +1,758 @@
+                {
+                    synthesis = new
+                    {
+                        video = new
+                        {
+                            protocol = new
+                            {
+                                name = "WebRTC",
+                                webrtcConfig = new
+                                {
+                                    clientDescription = localSdp,
+                                    iceServers = new[]
+                                    {
+                                        new
+                                        {
+                                            urls = new[] { firstUrl },
+                                            username = iceTokenObj!["Username"],
+                                            credential = iceTokenObj["Password"]
+                                        }
+                                    }
+                                }
+                            },
+                            format = new
+                            {
+                                crop = new
+                                {
+                                    topLeft = new
+                                    {
+                                        x = videoCrop.ToLower() == "true" ? 600 : 0,
+                                        y = 0
+                                    },
+                                    bottomRight = new
+                                    {
+                                        x = videoCrop.ToLower() == "true" ? 1320 : 1920,
+                                        y = 1080
+                                    }
+                                },
+                                bitrate = 1000000
+                            },
+                            talkingAvatar = new
+                            {
+                                customized = (isCustomAvatar?.ToLower() ?? "false") == "true",
+                                character = avatarCharacter,
+                                style = avatarStyle,
+                                background = new
+                                {
+                                    color = transparentBackground.ToLower() == "true" ? "#00FF00FF" : backgroundColor,
+                                    image = new
+                                    {
+                                        url = backgroundImageUrl
+                                    }
+                                }
+                            }
+                        }
+                    }
+                };
+
+                var connection = Connection.FromSpeechSynthesizer(speechSynthesizer);
+                connection.SetMessageProperty("speech.config", "context", JsonConvert.SerializeObject(avatarConfig));
+
+                var speechSynthesisResult = speechSynthesizer.SpeakTextAsync("").Result;
+                Console.WriteLine(speechSynthesisResult.ResultId);
```

Comment:
Please don't only print the id, give some prefix to make the message more meaningful.
e.g. Console.WriteLine($"Result ID: {speechSynthesisResult.ResultId}");

Reply:
updated